### ISSUE TITLE
Labels for moving MNIST

### ISSUE THREAD

**Issue by kirtyvedula** [praateekmahajan/moving-mnist-vae#1]
How do you get movingmnisttrain.npz and movingmnisttest.npz? I tried using the Gist file here - https://gist.github.com/praateekmahajan/b42ef0d295f528c986e2b3a0b31ec1fe

The issue is, it does not generate labels whereas the files you use assume that labels are generated in your code (train_loader does not have y)
